app.js

```diff
@@ -12,14 +12,19 @@ app.get("/api/articles", getArticles)
 
 app.get("/api/articles/:article_id/comments", getCommentById)
 
+app.use(express.json());
```

Comment:
Really good demonstration of TDD principles to see you include this _now that we need it_ to pass our test! 🥇